WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -230,11 +230,16 @@ private extension TotalsView {
 
     private var sendReceiptButton: some View {
         Button(action: {
-            isShowingSendReceiptModal = true
+            if posModel.eligibleWooCommerceVersionForPOSReceipts {
+                isShowingSendReceiptModal = true
+            } else {
+                isShowingReceiptsNotEligibleBanner = true
+            }
         }, label: {
             HStack(spacing: Constants.buttonSpacing) {
                 Text(Localization.sendReceipt)
                     .font(Constants.buttonFont)
+                    .transaction { $0.animation = nil }
```

Comment:
It doesn't solve animation issues for me, we can remove this and address animations later if you prefer.